pyspark.md

```diff
@@ -84,6 +116,45 @@ less memory fetching
 
 ---
 
+## How does __Arrow__ manage columnar data?
+
+---
+
+![fit](Images/Arrow_Table_i-0.png)
+
+^ Everything starts with a `Table` structure
+
+---
+
+![fit](Images/Arrow_Table_i-1.png)
+
+^ It is formed of `RecordBatches`, which contain a certain amount of rows. This
+makes it a streamable format
+
+---
+
+![fit](Images/Arrow_Table-0.png)
+
+---
+
+![fit](Images/Arrow_Table-1.png)
+
+---
+
+![fit](Images/Arrow_Table-2.png)
+
+^ Internally, `RecordBatches` have a columnar layout. Each `RecordBatch`
+contains some additional metadata.
+
+---
+
+![fit](Images/Arrow_Table-3.png)
+
+^ In the end, an Arrow `Table` is formed of a set of `RecordBatches`
```

Comment:
This is not really the case. A table can be more than just a collection of RecordBatches. For the talk it is probably best to keep this simplification. 

But as a bit of background knowledge:
A table is a collection of chunked arrays. These chunked array can consist of differently chunked pieces on a per column basis. When you create a table from a set of recordbatches, all columns are equally chunked.

Reply:
Hi @xhochy thanks! Yes, I know, but wanted to keep it as easy as possible... and RecordBatches are easier to relate to Pandas' blocks, so I decided to not go too deep and "lie" a bit, otherwise it can get too complicated for a relatively light talk. But I will add it as a "presenter note", which I use to add the technical parts I didn't want to add for clutter. Thanks a lot! (for the comment, having a look, and your work in Arrow!)